functions/billing/index.js

```diff
@@ -0,0 +1,156 @@
+/**
+ * Copyright 2018, Google LLC.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// [START functions_billing_limit]
+// [START functions_billing_stop]
+const {google} = require('googleapis');
+const {auth} = require('google-auth-library');
+
+const PROJECT_ID = process.env.GCP_PROJECT;
```

Comment:
Shouldn't the Project id be the id? I think GCP_PROJECT is not a variable we usually set

Reply:
It is [defined by GCF itself](https://cloud.google.com/functions/docs/env-var#reserved_keys_key_validation).